code/src/index.css

```diff
@@ -1,13 +1,398 @@
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	-webkit-font-smoothing: antialiased;
+	text-rendering: optimizeLegibility;
+	text-decoration: none;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+:root {
+  /* --primary: #ffffff;
+  --secondary: #fbcdd0;
+  --tertiary: #48466b;
+  --accent: #001053; */
+  --primary: #fefbfe;
+  --secondary: #121408;
+  --tertiary: #df9419;
+  --accent: #df9419;
+}
+
+body {
+  background-color: var(--secondary);
+  /* background-image: url('assets/susan-wilkinson-unsplash.jpg');
+  background-size: cover; */
+  /* background-image: url('assets/nathan-blackaby-unsplash.jpg');
+  background-size: cover; */
+}
+
+/* Header */
+header {
+  margin-bottom: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.logo {
+  width: 120px;
+  height: 120px;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+/* Text & Site-wide*/
+.underline {
+  text-decoration: underline;
+}
+
+.bold {
+  font-weight: 700;
+}
+
+.center {
+  text-align: center;
+}
+
+.error-message {
+  font-size: 20px;
+  text-decoration: underline;
+  padding: 20px 0 5px 0;
+  font-weight: 700;
+  text-align: center;
+}
+
+.explanation {
+  font-size: 17px;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 500;
+}
+
+.title {
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 700;
+  margin-bottom: 10px;
+}
+
+h2 {
+  padding-bottom: 10px;
+}
+
+p {
+  line-height: 1.5;
+}
+
+hr {
+  height: 2px;
+  border: 2px solid var(--tertiary);
+  border-radius: 2px;
+}
+
+.accordion-info a {
+  color: var(--secondary)
+}
+
+/* Page */
+.page {
+  min-width: 375px;
+  max-width: 80%;
+  min-height: 100vh;
+  margin: 0 auto;
+  font-family: 'Roboto', sans-serif;
+  position: relative;
+  padding-bottom: 3.5rem;
+}
+
+.page-container {
+  background-color: var(--primary);
+  color: var(--secondary);
+  border-radius: 5px;
+  padding: 15px;
+  margin: 10px 0;
+  border: 3px solid var(--accent);
+}
+
+/* Progress Bar */
+.progress-container {
+  width: 100%;
+  height: 20px;
+  border-radius: 5px;
+  background-color: var(--primary);
+  border: 3px solid var(--tertiary);
+}
+
+.progress-bar {
+  background-color: var(--tertiary);
+  transition: width 1s ease-in-out;
+  height: 100%;
+}
+
+/* Grid Display for Radio Buttons & Drop-down */
+.grid {
+  display: grid;
+  grid-template-columns: 2;
+  gap: 10px;
+  margin-top: 10px;
+}
+
+.column-1 {
+  grid-column: 1 / span 1;
+  padding: 5px 0;
+}
+
+.column-2 {
+  grid-column: 2 / span 1;
+  margin-left: auto;
+}
+
+/* Styling Text Input */
+input[type=text] {
+  width: 50%;
+  padding: 5px 10px;
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  border-radius: 5px;
+  border: 3px solid var(--secondary);
+}
+
+/* Styling Radio Buttons */
```

Comment:
Very impressed with the styling of the radio buttons, they are beautiful!

Reply:
Thank you! I'm still having some trouble with accessibility on these, so I need to go back and fix some things here to make them accessible.